lib/post_creator.rb

```diff
@@ -113,8 +113,9 @@ def valid?
 
       # Make sure none of the users have muted or ignored the creator or prevented
       # PMs from being sent to them
-      UserCommScreener.new(acting_user: @user, target_usernames: names).preventing_actor_communication.each do |username|
-        errors.add(:base, I18n.t(:not_accepting_pms, username: username))
+      target_users = User.where(username_lower: names.map(&:downcase)).pluck(:id, :username).to_u
```

Comment:
I think there is a typo here. `to_u` doesn't seem to be valid method to be called on an array.

Reply:
Heh specs would have picked that up, `u` is near `h` i guess :)